src/utils/formValidators.js

```diff
@@ -119,20 +125,43 @@ export const validateMARCWithElse = value => {
     `(((; else )(?=${quotedStringOrMarcPathPattern}))`,
     `((?<=(; else ))${quotedStringOrMarcPathPattern})))*$`].join(''));
 
-  if (!value || !value.length || value.match(pattern)) {
+  if (isEmpty(value) || value === allowedValue || value.match(pattern)) {
     return null;
   }
 
   return <FormattedMessage id="ui-data-import.validation.syntaxError" />;
 };
 
+/**
+ * Validate text inputs, check if they has remove option value
+ * validateTextFieldRemoveValue('###REMOVE###')
+ * // => null
+ * @param value
+ * @returns {null|*}
+ */
+export const validateTextFieldRemoveValue = value => {
+  if (isEmpty(value)) {
+    return null;
+  }
+
+  if (value.includes(REMOVE_OPTION_VALUE) && value !== REMOVE_OPTION_VALUE) {
```

Comment:
This will get you rid of the "if" condition above.
```suggestion
  if (!isEmpty(value) && value.includes(REMOVE_OPTION_VALUE) && value !== REMOVE_OPTION_VALUE) {
```

Reply:
Done